manager/notifier.go

```diff
@@ -56,6 +57,7 @@ var (
 	pagerdutyApiUrl        = flag.String("pagerdutyApiUrl", "https://events.pagerduty.com/generic/2010-04-15/create_event.json", "PagerDuty API URL.")
 	smtpSmartHost          = flag.String("smtpSmartHost", "", "Address of the smarthost to send all email notifications to.")
 	smtpSender             = flag.String("smtpSender", "alertmanager@example.org", "Sender email address to use in email notifications.")
+	hipchatApiUrl	       = flag.String("hipchatApiUrl","https://api.hipchat.com/v1/", "Hipchat API URL");
```

Comment:
This flag name is good for now, but I just noticed that this file is still using an old style of flag names and doesn't follow the convention we use everywhere else in Prometheus now (lower-case names with dots as namespace separators). So expect a flag name change here soon...